### PR TITLE
docs: update "install on k8s" guide to use helm repo instead of tarball.

### PR DESCRIPTION
**Pull request by ioga**
## Description

Recommend users to use helm repo instead of downloading our custom tarball.

## Test Plan

N/A

## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

[DET-123]: https://hpe-aiatscale.atlassian.net/browse/DET-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/setup-cluster/k8s/install-on-kubernetes.rst [hunk 1]
```diff
@@ -425,17 +459,20 @@ Determined <configure-determined-kubernetes-version>` to install on Kubernetes.
...
 
-To upgrade Determined or to change a configuration setting, first make the appropriate changes in
-``values.yaml`` and ``Chart.yaml``, and then run:
+To upgrade Determined or to change a configuration setting, update the helm repo, make the
+appropriate changes in ``values.yaml``, and then run:
 
 .. code::
 
-   helm upgrade <name for your deployment> --wait determined-helm-chart
+   helm repo update
```
Comment (dannysauer) [inline docs/setup-cluster/k8s/install-on-kubernetes.rst:467]: The sentence above says "update the helm repo ... and then run" , so either that or this `helm repo update` is redundant.

File: docs/setup-cluster/k8s/install-on-kubernetes.rst [hunk 2]
```diff
@@ -425,17 +459,20 @@ Determined <configure-determined-kubernetes-version>` to install on Kubernetes.
...
+   helm upgrade <name for your deployment> --wait determined-ai/determined
```
Comment (dannysauer) [inline docs/setup-cluster/k8s/install-on-kubernetes.rst:468]: This upgrade command should probably either include a `-f values.yaml` or drop the "edit values" from the instructions and include `--reuse-values`.

**Review by dannysauer — APPROVED** [review]
I'm good with this.  Thanks for writing it all up!

**Review by tara-det-ai — APPROVED** [review]
LGTM

**Pull request merged**